### ISSUE TITLE
checking my work

### ISSUE THREAD

**Issue by mmwwah** [scottkleinman/aeme#83]
Would you compare my encoding with this screenshot of the MS, to make sure that I've got the add/del/gap thing down?  I'd rather get it right now than have to go back and fix everything later, because there is a LOT more coming down the pipeline.  Thanks!

The MS:
![screen shot 2015-12-09 at 18 13 09](https://cloud.githubusercontent.com/assets/7809545/11701975/e6008c5c-9ea0-11e5-9890-c99719f91bea.png)

My code:
```    <l xml:id="junius1.394">&thornenl;att ga&thorn; o fo&wyn;&wyn;re &wyn;hele&sscap;&sscap;<pc rend=".">&middot;</pc><lb/>
                </l>
                <l xml:id="junius1.395"><pc>&PARAG;</pc>&ET; tu&slong;&slong; i&slong;&slong; <name type="person" ref="#bioJesus001"><choice>
                        <orig>c</orig>
                        <reg>Ch</reg>
                    </choice>ri&slong;t</name> <name type="person" ref="#bioAmminadab001">amminadab</name><pc>&middot;</pc> 
                </l>
                <l xml:id="junius1.396">&thorn;ur<choice>
                        <orig>&rsup;</orig>
                        <reg>r</reg>
                    </choice>h ga&slong;t<pc>&dash;</pc><lb/><add place="sublinear" hand="#bioOrm001">li&gins; &wyn;itt &gins;ehaten<choice>
                        <orig>&nsup;</orig>
                        <reg>n</reg>
                    </choice><pc>&punctelev;</pc></add><lb/>
                </l>
                <l xml:id="junius1.397"><gap quantity="1" unit="lines" reason="inked out"/><lb/>
                </l>
                <l xml:id="junius1.398">Forr &thorn;att he toc o rode d&aelig;&thorn;<pc>&middot;</pc><lb/>
                </l>```

**Comment by mmwwah** [comment]
Okay, I tried to add line breaks to make that code readable, but there must be a trick.  If you tell me what it is, I'll fix it so you don't have to read it as it is.

**Comment by scottkleinman** [comment]
I stuck it in oXygen and pretty printed it, so it's readable now.

**Comment by scottkleinman** [comment]
OK, the trick is to format your code using xml as follows:

<pre><code class="markdown">
```xml
&lt;l&gt;Your code here&lt;/l&gt;
```
</code></pre>

Onto your main question. I can see a couple of problems. In line 395, you've made a mistake that I've caught myself on several times. The spelling of "crist" should be "Crist" in the critical markup, not the modern "Christ". Also, "amminadab" should be given some critical markup with the name capitalised.

In terms of how you've handled `<gap>`, make sure that you used `inked-out` (with a hyphen). `inked out` implies two separate properties. I think there's possibly some ambiguity in the unit "lines" when applied to Junius 1. It might be useful to distinguish `lines` (of the manuscript) and `verse-lines`. But you're best positioned to judge that. The actual markup looks right.

**Comment by mmwwah** [comment]
1. Sorry about the slow reply -- I was away from my desk all day yesterday.

2. Thanks for reformatting.  Duly noted as to how to make the code readable.

3. Duly noted on hyphenating "inked-out."  I made that one up, after not finding anything useful in the TEI guidelines; is there a better option?  Does Laud 108 have any inked-out bits that I should conform to?

4.  I hadn't found anything in the AEME Guidelines and I decided to regularize "Christ" in the CritV.  The reason I'd settled on that is that I want the CritV to be student-usable (and not just seniors and grad students), so I am regularizing all the features that don't set the Ormulum apart, in order to highlight those features that do.  Should we discuss this further?

5. Ugh, thanks for catching the capitalization of Amminadab.  Dopey mistake.

6. Relieved that other than "inked[-]out," the handling of the correction seems okay.  Thank you for looking it over.

**Comment by scottkleinman** [comment]
The closest thing we have to the deletion in the image above is "linedThrough", but I can understand if you don't think that's adequately descriptive. I'm not sure that we can set css to make distinctions between different kinds of deletions in the html output; more likely, we could mark a deletion and give access to the `@reason` value in a tooltip or something like that. So what you use doesn't matter, as long as you think it is clear. *That said*, when I try `gap@reason`, I get a validation error. We probably need to update the schema. I'll post that as a separate issue. For now, you may just have to put the attributes in an xml comment and go back and add them when we update the schema.

Changing "crist" to "Crist" in the critical view changes the word to modern capitalisation but does not change it to a modern spelling (making a distinction between capitalisation and spelling for the purposes of this argument). Changing it to "Christ" modernises the spelling as well as the capitalisation. Logically, you would then also have to modernise all of Orm's spellings: e.g. "godd" would become "God". Obviously, that's not a logic you can follow. You may have reasons for making in exception for Christ, e.g. for the pedagogical reasons you give. But it's probably an exception that you'll have to justify somewhere in explanatory notes.

**Comment by mmwwah** [comment]
Yeah (re Crist vs Christ) that's a good point.  In any case, it's something that I can do with a single search&replace, so I'm not going to sweat it now.

**Comment by scottkleinman** [comment]
Just be careful that you don't have "Christ" in any Modern English notes! :-)

**Comment by dorothyk98** [comment]
Since Orm does the "inked-out" enough, should we just made it a visual render plus note? Or is there a reason to come out with a specific tag so that it can be searched later? Also, it sounds like at some point when Meg has worked her way through that it makes sense to go back to the AEME guidelines and do an update.

**Comment by mmwwah** [comment]
Yes to update.  I'm also putting a lot of stuff in the critical apparatus/intro that I'm writing for tenure purposes.

Orm has two different styles of marking out: one legible and one completely omg-don't-read-this.  The former is what I'm saving reason = overlined for.
